components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Extensions/PublishedDataSetEventsModelEx.cs

```diff
@@ -20,7 +20,7 @@ public static class PublishedDataSetEventsModelEx {
         /// <param name="displayName"></param>
         /// <returns></returns>
         public static MonitoredItemModel ToMonitoredItem(
-            this PublishedDataSetEventsModel publishedEvents,
+            this PublishedDataSetEventModel publishedEvents,
```

Comment:
if you rename to singular, you should also rename the parameter

Reply:
Good catch, will fix.